src/api/app/controllers/webui/obs_factory/staging_projects_controller.rb

```diff
@@ -37,13 +37,16 @@ def show
         format.html do
           # FIXME: For staging repositories only the images repository is relevant atm
           # However, we should make this configurable in the future
-          images_repository = @staging_project.repositories.find_by(name: 'image')
+          images_repository = @staging_project.repositories.find_by(name: 'images')
```

Comment:
This was probably a bug we didn't know about since the name of repositories for images is plural.

Reply:
wouldn't it be a good idea to mention this in the commit message? :thinking:

Reply:
The message says already:

```
it is images and not image. This caused the the display of repository checks to fail.
```

What is missing? I think it is pretty clear that this was a spelling mistake...